compiler/tests/evm_contracts.rs

```diff
@@ -188,6 +189,27 @@ fn return_u256() {
     })
 }
 
+
+#[rstest(fixture_file, input, expected,
+    case("return_addition_u256.vy", vec![42, 42], Some(84)),
+    case("return_subtraction_u256.vy", vec![42, 42], Some(0)),
+    case("return_multiplication_u256.vy", vec![42, 42], Some(1764)),
+    case("return_division_u256.vy", vec![42, 42], Some(1)),
```

Comment:
There's stuff like division by zero that we could test here but I don't want to open a can of worms and put it in an issue for now https://github.com/ethereum/fe/issues/41

Reply:
Yeah, for now I think it's ok to put things like this off. Below are some thoughts on how I think this should be done.

My thinking right now is that we will introduce *safe* math functions to the runtime, where we check for things like overflow and div-by-zero. Then instead of mapping `a / b` to `div(a', b')`, we map to `safe_div(a', b')`, where safe_div is a function that we've defined that reverts on a zero denominator.

I think the semantic analysis pass should be responsible for determining what functions are needed during runtime and then we'll include them in the Yul contract when mapping from the Fe contract. This approach would be extended to things like maps where we have generic type params. For example, defining the contract field `pub guest_book: map<address, bytes[100]>` would result in two function being added to the runtime named something like *map_get_address_bytes100* and *map_insert_address_bytes100*, which would be used in place of the expression `self.guest_book[0x00]` and statement `self.guest_book[0x00] = some_bytes`.

Reply:
Great feedback, I put it on #41